packages/babylon/test/fixtures/esprima/invalid-syntax/migrated_0065/options.json

```diff
@@ -1,3 +1,3 @@
 {
-  "throws": "Unexpected character '#' (1:2)"
-}
\ No newline at end of file
+  "throws": "Unexpected token (1:2)"
```

Comment:
I’m not sure why I changed this. I might have been unable to preserve while modifying the tokenizer to distinguish between `#identifier` private fields and `#` topic reference. I’ll try to change the tokenizer more, so that it preserves the original error message here.